### ISSUE TITLE
Maxwell does not start on fresh ec2 instance

### ISSUE THREAD

**Issue by jonathanvx** [zendesk/maxwell#436]
So I was testing out maxwell on a fresh ec2 instance. No matter what I tried (which wasn't much), I kept getting: 
`[ec2-user@ip-172-30-0-96 maxwell-1.2.1]$ ./bin/maxwell --user='root'  --host='127.0.0.1' --producer=stdout `

```
09:48:03,487 INFO  Maxwell - Maxwell v1.2.1 is booting (StdoutProducer), starting at BinlogPosition[binlog.000001:107]
java.lang.NullPointerException
	at com.google.code.or.net.impl.AuthenticatorImpl.login(AuthenticatorImpl.java:73)
	at com.google.code.or.net.impl.TransportImpl.connect(TransportImpl.java:105)
	at com.google.code.or.OpenReplicator.start(OpenReplicator.java:106)
	at com.zendesk.maxwell.MaxwellReplicator.startReplicator(MaxwellReplicator.java:127)
	at com.zendesk.maxwell.MaxwellReplicator.beforeStart(MaxwellReplicator.java:142)
	at com.zendesk.maxwell.RunLoopProcess.runLoop(RunLoopProcess.java:29)
	at com.zendesk.maxwell.Maxwell.start(Maxwell.java:163)
	at com.zendesk.maxwell.Maxwell.main(Maxwell.java:180)
```

Am I doing something wrong?

**Comment by osheroff** [comment]
you need to provide a password.  I, apparently, need to check for null passwords properly.

**Comment by osheroff** [comment]
fixed in 1.3.0
